packages/runtime/container-runtime/src/test/layerCompatValidation.spec.ts

```diff
@@ -1,294 +0,0 @@
-/*!
```

Comment:
This file is renamed to `runtimeLayerCompatValidation.spec.ts` and tests for data store layer validation is added to it.